test/e2e/bots_test.go

```diff
@@ -264,6 +264,19 @@ func runBotTest(t *testing.T,
 			err := botDriver.WaitForLastMessageEqual(botDriver.BotUserID(), botDriver.Channel().ID(), expectedMessage)
 			assert.NoError(t, err)
 		})
+
+		t.Run("Echo Executor help", func(t *testing.T) {
+			command := "echo help"
+			expectedMessage := ".... empty response _*&lt;cricket sounds&gt;*_ :cricket: :cricket: :cricket:"
+			if botDriver.Type() == DiscordBot {
+				expectedMessage = ".... empty response _*<cricket sounds>*_ :cricket: :cricket: :cricket:"
+			}
+
+			botDriver.PostMessageToBot(t, botDriver.Channel().Identifier(), command)
+			err := botDriver.WaitForLastMessageEqual(botDriver.BotUserID(), botDriver.Channel().ID(), expectedMessage)
+			assert.NoError(t, err)
+		})
+
```

Comment:
WDYT about adding also `helm help` to the E2E tests? In that way we'd test both cases (empty help + provided help)  🤔